modules/client-configuration/pages/clients-pubcloud.adoc

```diff
@@ -97,6 +97,9 @@ When the bootstrap process has completed successfully, your client is listed at
 * When registering clients launched from a public cloud image (for example, AWS AMI), you need to do some additional configuration to prevent them from over-writing each other.
   For more information about registering clones, see xref:administration:troubleshooting/tshoot-registerclones.adoc[].
 
+// 2023-05-31, ke: using snippet here?
```

Comment:
Do not leave a comment. This needs to be  decided. Either we include it, either we do not :-D

Reply:
ok, I do not include it.